Data/DbInitializer.cs

```diff
@@ -72,12 +72,56 @@ public static void Initialize(IServiceProvider serviceProvider)
               }
               context.SaveChanges();
 
+              var productSubTypes =  new ProductSubType[]
+              {
+                  //1
+                  new ProductSubType {
+                      Label = "Portable Electronics",
+                      ProductTypeId = 1
+                  },
+
+                  //2
+                  new ProductSubType {
+                      Label = "Video Game Systems",
+                      ProductTypeId = 1
+                  },
+
+                  //3
+                  new ProductSubType {
+                      Label = "Kitchen Appliances",
+                      ProductTypeId = 2
+                  },
+
+                  //4
+                  new ProductSubType {
+                      Label = "Yard Equipment",
+                      ProductTypeId = 2
+                  },
+
+                  //5
+                  new ProductSubType {
+                      Label = "Living Room",
+                      ProductTypeId = 3
+                  },
+
+                  //6
+                  new ProductSubType {
+                      Label = "Bedroom",
+                      ProductTypeId = 3
+                  }
+              };
+
+             foreach (ProductSubType i in productSubTypes)
+              {
+                  context.ProductSubType.Add(i);
+              }
 
```

Comment:
add `context.saveChanges();` here after adding them to the context. They need to be available as foreign keys for the new products below.